src/main/java/com/questhelper/steps/ObjectStep.java

```diff
@@ -232,14 +232,16 @@ public void onGameStateChanged(GameStateChanged event)
 		}
 	}
 
-	public void addAlternateObjects(Integer... alternateObjectIDs)
+	public QuestStep addAlternateObjects(Integer... alternateObjectIDs)
```

Comment:
If it's not an overridden function, we should probably return `ObjectStep` instead of `QuestStep`